tests/phpunit/tests/test-filters.php

```diff
@@ -363,4 +363,37 @@ public function test_get_terms_inside_query() {
 		add_action( 'pre_get_posts', array( $this, '_action_pre_get_posts' ) );
 		get_posts(); // fires the action and performs the assertions.
 	}
+
+	/**
+	 * @ticket #1682 and Polylang for WooCommerce #939
+	 *
+	 * @see https://github.com/polylang/polylang/issues/1682
+	 * @See https://github.com/polylang/polylang-wc/issues/939
+	 */
+	public function test_comments_clauses() {
```

Comment:
```suggestion
	public function test_db_error_in_comments_clauses() {
```

Reply:
Done in 6c72c0e8bfa5ad5d86369297b3395c41ecf1448d